CONTRIBUTING.md

```diff
@@ -1,15 +1,60 @@
-# Guidelines for contributing code
+# Guidelines for Contributing Code
 
 New Relic welcomes code contributions by the Node community to this module, and
 have taken effort to make this process easy for both contributors and our
 development team.
 
+## Process
+
+### Feature Requests
+
+Feature requests should be submitted in the [Issue tracker](../../issues), with
+a description of the expected behavior & use case. Before submitting an Issue,
+please search for similar ones in the [closed
+issues](../../issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement).
+
+### Pull Requests
+
+We can only accept PRs for version v3.0.1 or greater due to open source
```

Comment:
Believe this will be `v3.1.0` if we stick with doing license change as a minor version.

Reply:
Updated